lms/djangoapps/instructor_task/tasks_helper.py

```diff
@@ -586,12 +586,12 @@ def upload_grades_csv(_xmodule_instance_args, _entry_id, course_id, _task_input,
             cohorts_group_name = []
             if course.is_cohorted:
                 group = get_cohort(student, course_id, assign=False)
-                cohorts_group_name.append(group.name if group else '')
+                cohorts_group_name.append(unicode(group.name) if group else u'')
```

Comment:
`get_cohort` returns a `CourseUserGroup`, which is a django model using a `CharField`, which according to https://docs.djangoproject.com/en/1.4/ref/unicode/#models is always going to return a `unicode`.